src/mixins/amalgamation-mixin.ts

```diff
@@ -356,13 +368,13 @@ export default class AmalgamationMixin extends Vue {
    * @param isNew Whether the business is new (ie, set by user, not restored from draft).
    */
   async updatePrepopulatedData (business: AmalgamatingBusinessIF, isNew = false): Promise<void> {
-    // safety check
-    if (!business || business.type !== AmlTypes.LEAR) {
-      throw new Error('updatePrepopulatedData(): invalid business')
-    }
+    // safety checks
+    if (!business || business.type !== AmlTypes.LEAR) throw new Error('Invalid business')
+    if (!business.addresses) throw new Error('Missing business addresses')
+    if (!business.authInfo) throw new Error('Missing share classes')
```

Comment:
This is to display a console error (and however the caller wants to handle the exception) in case there is any error with the data (eg, Riyaz's testing in [19750](https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/19750)).